feature-utils/poly-import/src/entities/processed-data.js

```diff
@@ -0,0 +1,31 @@
+export default class ProcessedData {
+    constructor() {
+        let self = this;
+        this._importersData = null;
+        this._unknownFolderNames = null;
+
+        this._missingKnownFileNames = null;
+        this._missingCommonFileNames = null;
+
+        this._reportMetadata = null;
+
+        const attributes = [
+            "importersData",
+            "unknownFolderNames",
+            "missingKnownFileNames",
+            "missingCommonFileNames",
+            "reportMetadata",
+        ];
+        //setters & getters
+        attributes.forEach(function (attribute) {
```

Comment:
You could use this loop to also create the attributes. The way you're doing it now, you're repeating the names in two different places.